pyproject.toml

```
@@ -0,0 +1,236 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "aleph-sdk-python"
+dynamic = ["version"]
+description = "Lightweight Python Client library for the Aleph.im network"
+readme = "README.md"
+license = { file = "LICENSE.txt" }
+authors = [
+    { name = "Aleph.im Team", email = "hello@aleph.im" },
+]
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Development Status :: 4 - Beta",
+    "Framework :: aiohttp",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: POSIX :: Linux",
+    "Operating System :: MacOS :: MacOS X",
+    "Topic :: Software Development :: Libraries",
+]
+dependencies = [
+    "aiohttp>=3.8.3",
+    "aleph-message~=0.4.3",
+    "coincurve; python_version<\"3.11\"",
+    "coincurve>=17.0.0; python_version>=\"3.11\"",
+    "eth_abi>=4.0.0; python_version>=\"3.11\"",
+    "eth_account>=0.4.0",
+    "python-magic",
+    "typer",
+    "typing_extensions",
+]
+
+[project.optional-dependencies]
+cosmos = [
+    "cosmospy",
+]
+dns = [
+    "aiodns",
+]
+docs = [
+    "sphinxcontrib-plantuml",
+]
+ethereum = [
+    "eth_abi>=4.0.0; python_version>=\"3.11\"",
+    "eth_account>=0.4.0",
+]
+ledger = [
+    "ledgereth==0.9.0",
+]
+mqtt = [
+    "aiomqtt<=0.1.3",
+    "certifi",
+    "Click",
+]
+nuls2 = [
+    "aleph-nuls2",
+]
+polkadot = [
+    "py-sr25519-bindings",
+    "substrate-interface",
+]
+solana = [
+    "base58",
+    "pynacl",
+]
+tezos = [
+    "aleph-pytezos==0.1.1",
+    "pynacl",
+]
+encryption = [
+    "eciespy; python_version<\"3.11\"",
+    "eciespy>=0.3.13; python_version>=\"3.11\"",
+]
+all = [
+    "aleph-sdk-python[cosmos,dns,docs,ethereum,ledger,mqtt,nuls2,polkadot,solana,tezos,encryption]",
+]
+
+[project.urls]
+Documentation = "https://aleph.im/"
+Homepage = "https://github.com/aleph-im/aleph-sdk-python"
+
+[tool.hatch.version]
+source = "vcs"
+
+[tool.hatch.build.targets.wheel]
+packages = [
+    "src/aleph",
+    "pyproject.toml",
+    "README.md",
+    "LICENSE.txt",
+]
+
+[tool.hatch.build.targets.sdist]
+include = [
+    "src/aleph",
+    "pyproject.toml",
+    "README.md",
+    "LICENSE.txt",
+]
+
+[tool.isort]
+profile = "black"
+
+[tool.hatch.envs.test]
+features = [
+    "cosmos",
+    "dns",
+    "ethereum",
+    "ledger",
+    "nuls2",
+    "polkadot",
+    "solana",
+    "tezos",
+    "encryption",
+]
+dependencies = [
+    "pytest==8.0.1",
+    "pytest-cov==4.1.0",
+    "pytest-mock==3.12.0",
+    "pytest-asyncio==0.23.5",
+    "fastapi",
+    "httpx",
+    "secp256k1",
+]
+[tool.hatch.envs.test.scripts]
+run-coverage = "pytest --cov-config=pyproject.toml --cov=pkg --cov=tests"
+run = "run-coverage --no-cov {args:.}"
```

Why not just `pytest` ?
```suggestion
run = "pytest {args:.}"
```